src/test/java/api/item/ItemStatusApiTests.java

```diff
@@ -0,0 +1,73 @@
+package api.item;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.CoreMatchers.notNullValue;
+
+import api.support.APITests;
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+public class ItemStatusApiTests extends APITests {
+
+  private static final String ITEM_STATUS = "status";
+  private static final String ITEM_STATUS_DATE = "date";
+  private static final String ITEM_BARCODE = "barcode";
+
+  @Test
+  public void itemStatusDateShouldExistsAfterCheckout() throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    IndividualResource user = usersFixture.jessica();
+
+    loansFixture.checkOutByBarcode(item, user, new DateTime(DateTimeZone.UTC));
+
+    JsonObject checkedOutItem = itemsClient.get(item.getId()).getJson();
+
+    assertThat(checkedOutItem.getJsonObject(ITEM_STATUS)
+        .getString(ITEM_STATUS_DATE), is(notNullValue()));
+  }
+
+  @Test
+  public void itemStatusDateShouldNotChangedIfItemChanged() throws InterruptedException,
```

Comment:
Is this behaviour defined by mod-circulation?

My interpretation is that this is effectively testing that the fake storage module behaves as it is expected to?

Reply:
@marcjohnson-kint Thank you! I got it. This test is redundant in mod-circulation.

 I removed it.